apps/backend/src/user/user.service.ts

```diff
@@ -1,4 +1,33 @@
 import { Injectable } from '@nestjs/common';
+import { PrismaService } from 'nestjs-prisma';
+
+import { CreateUserDto } from './dto/create-user.dto';
+import { UpdateUserDto } from './dto/update-user.dto';
+import { UserDto } from './dto/user.dto';
 
 @Injectable()
-export class UserService {}
+export class UserService {
+  constructor(private readonly prisma: PrismaService) {}
+
+  async findOne(id: string): Promise<UserDto> {
+    return this.prisma.user.findUnique({
+      where: { authSchId: id },
+    });
+  }
```

Comment:
this will return null if the user is not found, best to throw a notfound exception